static/app/views/replays/detail/console/consoleMessage.tsx

```diff
@@ -45,35 +49,45 @@ function MessageFormatter({breadcrumb}: MessageFormatterProps) {
     ? sprintf.parse(message).filter(parsed => Array.isArray(parsed))
     : [];
 
-  // TODO `%c` is console specific, it applies colors to messages
-  // for now we are stripping it as this is potentially risky to implement due to xss
-  const consoleColorPlaceholderIndexes = placeholders
-    .filter(([placeholder]) => placeholder === '%c')
-    .map((_, i) => i);
-
-  // Retrieve message formatting args
-  const messageArgs = args.slice(0, placeholders.length);
-
-  // Filter out args that were for %c
-  for (const colorIndex of consoleColorPlaceholderIndexes) {
-    messageArgs.splice(colorIndex, 1);
+  if (placeholders.length) {
```

Comment:
I'm not positive what's going on with the data we're using, but instead of checking placeholders length, I think a safer option would be to handle this case more specifically. Let's run this case when `breadcrumb.message !== message` (I would generally expect `breadcrumb.message` to equal `breadcrumb.data.arguments[0]` and I'm not sure why this wasn't the case here).

We should also add a comment about it.